Laravel-whib/database/migrations/2023_11_11_190245_create_map_pins_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('map_pins', function (Blueprint $table) {
+            $table->id();
+            $table->string('pin_name');
+            $table->string('description')->nullable();
+            $table->boolean('favourite');
+            $table->decimal('latitude', 10, 8);
+            $table->decimal('longitude', 11, 8);
+            $table->string('user_id');
```

Comment:
This is not foreign key. Look up on laravel site how foreign key looks like. https://laravel.com/docs/10.x/migrations#foreign-key-constraints